eng/common/scripts/Delete-RemoteBranches.ps1

```diff
@@ -50,12 +71,37 @@ foreach ($branch in $branches)
     continue
   }
 
-  LogDebug "Branch [ $branchName ] in repo [ $RepoName ] has no associated open Pull Request. Deleting Branch"
-  try{
-    Remove-GitHubSourceReferences -RepoOwner $RepoOwner -RepoName $RepoName -Ref ($branch.Remove(0,5)) -AuthToken $AuthToken
-  }
-  catch {
-    LogError "Remove-GitHubSourceReferences failed with exception:`n$_"
-    exit 1
+  LogDebug "Branch [ $branchName ] in repo [ $RepoName ] has no associated open Pull Request."
+  # Allow to override the the var $LastCommitOlderThan
+  if ($LastCommitOlderThan -or $env:LASTCOMMITDATEOVERRIDE) {
```

Comment:
Why is the override env variable interesting?

Reply:
Just for testing purpose. Avoid using the different date for different build

Reply:
Ok lets remove that and just pass in different parameters in the script when testing locally.